### DISCUSSION TITLE
Konfiguration Watchtower

### DISCUSSION THREAD

**Discussion by mmuegge** [solectrus/solectrus#3724] [Q&A]
Hallo,
ich möchte gerne im Solectrus Dockersystem Watchtower als Service einrichten.
In der Beschreibung steht, dass in der Datei compose.yaml der Watchtower als Service eingetragen wird.
Auf meinem System gibt es aber nur die Datei docker-compose.yml was auf eine ältere Dockerversion hinweist.
Muss Docker Compose upgedatet werden, wenn ja wie oder kann ich den Watchtower-Dienst in der Datei docker-compose.yml eintragen?

**Comment by ledermann** [comment]
Deine Docker-Version ist vermutlich gar nicht älter, sondern du hast SOLECTRUS vor längerer Zeit installiert, als der Dateinamen noch `docker-compose.yaml` hieß. Ob der Dateiname `compose.yaml` oder `docker-compose.yml` heißt, ist heute unerheblich, siehe hier:
https://docs.solectrus.de/support/docker#und-was-ist-docker-compose

Ändere den Dateinamen also am besten gar nicht, sondern ergänze nur den Watchtower-Service gemäß der Anleitung: https://docs.solectrus.de/referenz/watchtower/konfiguration
